include/seastar/core/internal/estimated_histogram.hh

```diff
@@ -326,12 +327,21 @@ public:
         return _buckets[b];
     }
 
+    size_t get_schema() const noexcept {
+            return PRECISION_BITS;
+    }
+
+    size_t min_as_spars_id() const noexcept {
```

Comment:
could you explain what "spars" stands for?

Reply:
renamed to make it clearer